.eslintrc.base.json

```diff
@@ -23,7 +23,6 @@
 	"reportUnusedDisableDirectives": true,
 	"root": true,
 	"rules": {
-		"linebreak-style": ["error", "unix"],
```

Comment:
@eamodio just a heads up about removing this rule since it isn't cross-plat friendly.